index.html

```diff
@@ -4235,29 +4195,32 @@ <h2>DID URL Dereferencing</h2>
     </p>
 
       <section>
-        <h3>DID URL Dereferencing Input Metadata Properties</h3>
+        <h3>DID URL Dereferencing Options</h3>
 
         <p>
 The possible properties within this structure and their possible values SHOULD
 be registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
-This specification defines the following common properties.
+This specification defines the following common properties. This
+specification defines the following common properties for dereferencing
+options:
             </p>
 
             <dl>
-                <dt>
+              <dt>
 accept
-                </dt>
-                <dd>
-The MIME type the caller prefers for <code>content-stream</code>. The <a>DID
-URL dereferencing</a> implementation SHOULD use this value to determine the
-<a>representation</a> contained in the returned value if such a <a>representation</a> is
-supported and available.
-                </dd>
+              </dt>
+              <dd>
+The MIME type that the caller prefers for <code>contentStream</code>. The  MIME
+type MUST be expressed as an <a data-lt="ascii string">ASCII string</a>.
+The <a>DID URL dereferencing</a> implementation SHOULD use this value to
+determine the <a>representation</a> contained in the returned value if such a
```

Comment:
I think we may mean `contentType` of the `resource`... and not want to use `representation` here.

Reply:
Fixed a while ago.